README.md

```diff
@@ -21,17 +21,21 @@ The easiest way to get started is using Docker:
 
 ```bash
 # Build the image
-./scripts/build.sh
+docker build -t mcp-server-troubleshoot:latest .
 
 # Run the server
-./scripts/run.sh /path/to/bundles your-token
+docker run -i --rm \
+  -v "/path/to/bundles:/data/bundles" \
+  -e SBCTL_TOKEN="your-token" \
+  -e MCP_BUNDLE_STORAGE="/data/bundles" \
```

Comment:
Again I don't see any reason to document this ENV anywhere in documentation it's a development artifact.